FGEGraphics/UISystem/UIPositionHelper.cs

```diff
@@ -138,6 +147,11 @@ public UIPositionHelper ConstantXY(int x, int y)
         return this;
     }
 
+    /// <summary>Sets a constant X and Y value using a <see cref="Location"/>.</summary>
+    /// <param name="location">The location.</param>
```

Comment:
Doc standards have shifted since this code was originally written - I had a lot of docs like this but came to realize docs that say nothing ("location" = "the location"?) are worse than no docs, so it's preferred to exclude them if there's no actual information to give